Check

Ch-AhteshamUlHaq opened the pull request:
Created the repository Named "Portfolio-Mobile-Version"
Created the Branch and then added the folder in the branch named ".github" with sub folder "workflows"
Created the linter.yml file in the subfolder "workflows". and paste the code in that.
Created the index.html file and develop the front-end
Created the style.css file and styled the front-end using Grid, Flexbox, Hover and etc.
It is developed responsive for desktop and mobile versions both.
created the "test.md" file.
Created the ".gitignore" and added in that "node module/" and "test.md"
Added the Readme file and followed the microverse suggested template.

Ch-AhteshamUlHaq committed:
Created The linters.yml file

Ch-AhteshamUlHaq committed:
Created the .gitignore file in the root directory

Ch-AhteshamUlHaq committed:
Added about me png

Ch-AhteshamUlHaq committed:
Added PNG image cap

Ch-AhteshamUlHaq committed:
Created the index.html file

Ch-AhteshamUlHaq committed:
created the style.css file

Ch-AhteshamUlHaq committed:
added the arrow.png

Ch-AhteshamUlHaq committed:
added css icon.png

Ch-AhteshamUlHaq committed:
added the counter.png

Ch-AhteshamUlHaq committed:
Added the Html png

Ch-AhteshamUlHaq committed:
Added javascript png

Ch-AhteshamUlHaq committed:
added git icon

Ch-AhteshamUlHaq committed:
Added Lindin icon

Ch-AhteshamUlHaq committed:
Added the Logo.png

Ch-AhteshamUlHaq committed:
added the png

Ch-AhteshamUlHaq committed:
added the the background png

Ch-AhteshamUlHaq committed:
Added the menu icon png file

Ch-AhteshamUlHaq committed:
Added project Two image

Ch-AhteshamUlHaq committed:
added the icon

Ch-AhteshamUlHaq committed:
Added the project Four image

Ch-AhteshamUlHaq committed:
Added the project One image

Ch-AhteshamUlHaq committed:
Added the project Three image

Ch-AhteshamUlHaq committed:
Added the twitter icon

Ch-AhteshamUlHaq committed:
added the Vector icon

Ch-AhteshamUlHaq committed:
check

Ch-AhteshamUlHaq committed:
Updated the index.html file

Ch-AhteshamUlHaq committed:
Updated the readme file

Ch-AhteshamUlHaq committed:
Updated the style.css

Ch-AhteshamUlHaq committed:
Remove the linter errors

Ch-AhteshamUlHaq committed:
Again remove linter errors

Ch-AhteshamUlHaq committed:
Removed the extra spaces

Ch-AhteshamUlHaq committed:
Updated

Ch-AhteshamUlHaq committed:
Updated again

Ch-AhteshamUlHaq committed:
Tried again

Ch-AhteshamUlHaq committed:
Added .hintrc file

Ch-AhteshamUlHaq committed:
added .stylelintrc.json

Ch-AhteshamUlHaq committed:
added package json

Ch-AhteshamUlHaq committed:
added package json file

Ch-AhteshamUlHaq committed:
Updated with linter suggestions

Ch-AhteshamUlHaq committed:
Totally stressed now

Ch-AhteshamUlHaq committed:
Updated

Ch-AhteshamUlHaq committed:
Tried again

Ch-AhteshamUlHaq committed:
Removed linter errors

Ch-AhteshamUlHaq committed:
Updated

Ch-AhteshamUlHaq committed:
Updated

Ch-AhteshamUlHaq committed:
At last removed all lint errors

Ch-AhteshamUlHaq committed:
Finaly succeded Removed all linter errors

Ch-AhteshamUlHaq committed:
Updated

Ch-AhteshamUlHaq committed:
Yay! It's done now

Ch-AhteshamUlHaq committed:
All spaces are good now!

Kossi-stack reviewed (changes requested):
## Hi @Ch-AhteshamUlHaq,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966](https://user-images.githubusercontent.com/81444338/170121142-a9f93bb7-38bb-45cc-8327-aab2501ef90a.gif)

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

## Required Changes ♻️

 - [x] Kindly edit the PR's title to something more descriptive. it can be "Portfolio: mobile version Project PR or Portfolio: mobile version PR".
 - [x] Please Portfolio: mobile version sure to use [this readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) and please be sure to edit all of the sections to better suit your project.
- [x] Kindly make sure in your next PR to use a  meaningful commit
- [x] Kindly remove the package.json file in the .gitignore file.

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

Ch-AhteshamUlHaq committed:
Removed the package.json from the .gitignore file.

Ch-AhteshamUlHaq committed:
Changed the Title in index.html

Ch-AhteshamUlHaq committed:
Updated the Readme file in a meaningful way.

Ch-AhteshamUlHaq commented:
Edited the PR's as "Portfolio: Mobile Version Projects PR"
Added the description in the Readme.md as required.
Made the meaningful commits.
Removed the package.json file from the .gitignore file.

muneebulrehman commented on README.md:
```diff
@@ -1,2 +1,34 @@
-# Portfolio-Mobile-Version-
-Setup a mobile version portfolio to display my projects and skills
+# Portfolio-Mobile-Version
+Setup a mobile version Portfolio: Mobile Version Projects PR to display my projects and skills
+
+Hi, I simple coded the mobile version with the help of flexbox and figma features. It is a structure where we decorate and furnishe further. Develop the responsive one, It's a Portfolio: Mobile Version projects display. It helps the users to understand our services better. It also helps users to understand now a days features and techs.
+
+Built With Major languages Frameworks Technologies used Html, Css, linters.
+Grid,
+Flexbox,
+Hover
+
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites 
+Setup 
+Install 
+Usage 
+Run 
+tests
+Deployment 
+Authors 
+👤 Author
+
+GitHub: @ch-ahtesham-ul-haq Twitter: @chaury39 
+LinkedIn: https://www.linkedin.com/in/ch-ahtesham-ul-haq-b5318b152/
+
+🤝 Contributing Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support Give a ⭐️ if you like this project!
+
+Acknowledgments Hat tip to anyone whose code was used Inspiration etc 
+📝 License
+ This project is MIT licensed.
```
Hi, even though you used the content from the provided README template, you didn't use the markdown syntax to style the file. Kindly update it with the same. If you have any difficulty, refer to [this](https://www.markdownguide.org/cheat-sheet/) article

Ch-AhteshamUlHaq replied:
Okay Thank You So Much

muneebulrehman reviewed (changes requested):
Hi @Ch-AhteshamUlHaq , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Good Pr description
- Good follow-up with the previous reviewer.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Ch-AhteshamUlHaq committed:
Updated as suggested by reviewer.

algerina commented on README.md:
```diff
@@ -1,2 +1,38 @@
-# Portfolio-Mobile-Version-
-Setup a mobile version portfolio to display my projects and skills
+# Portfolio-Mobile-Version
+
+<h1>Setup a mobile version Portfolio: Mobile Version Projects PR to display my projects and skills</h1>
+
+<h2>Hi, I simple coded the mobile version with the help of flexbox and figma features.
+It is a structure where we decorate and furnishe further. Develop the responsive one,
+It's a Portfolio: Mobile Version projects display. It helps the users to understand 
+our services better. It also helps users to understand now a days features and techs. </h2>
+
+<h3>Built With Major languages Frameworks Technologies used Html, Css, linters.</h3>
+<h4>Grid,</h4>
+<h4>Flexbox,</h4>
+<h4>Hover</h4>
+
+<h5>To get a local copy up and running follow these simple example steps.</h5>
+
+Prerequisites 
+Setup 
+Install 
+Usage 
+Run 
+Tests
+Deployment 
+Authors 
+👤 Author
+
+GitHub: @ch-ahtesham-ul-haq Twitter: @chaury39 
+LinkedIn: https://www.linkedin.com/in/ch-ahtesham-ul-haq-b5318b152/
+
+🤝 Contributing Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support Give a ⭐️ if you like this project!
+
+Acknowledgments Hat tip to anyone whose code was used Inspiration etc 
+📝 <h6>License</h6>
+ This project is MIT licensed.
```
- [x] You did a great job, however, your Readme file still doesn't fit Microverse template. Kindly use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) as it  is. Just remove the sections that you don't need and add your personal informations. here is an [example.](https://github.com/algerina/My_portfolio/blob/main/README.md)

Ch-AhteshamUlHaq replied:
Followed the template. Thank You

algerina reviewed (changes requested):
Hi @Ch-AhteshamUlHaq , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media2.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Highlights
- You did a great job implementing most of the requested changes.
- You have a meaningful PR desription

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Ch-AhteshamUlHaq committed:
Changed as was suggested by the reviwer

sinansevgi reviewed (approved):
Hi @Ch-AhteshamUlHaq, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Readme has all main titles  now :+1: 

## Optional suggestions
- [ ] It will be great if you can use [`markdown`](https://www.markdownguide.org/cheat-sheet/) syntax in the readme file instead of `HTML` syntax. Readme files are markdown files and embedded HTML doesn't work well. You can make your readme more readable with using markdown syntax

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Happy Hacking! :computer: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
Sinan Sevgi

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
